www/js/model.js

```diff
@@ -311,6 +336,15 @@ define([
 				console.log("node added", newNode.ID, newNode.type);
 				return newNode.ID;
 			},
+			updatePositionXY: function(/*string */ id){
+				console.log(obj);
```

Comment:
please add some comments for this one as well. I remember you mentioning it why you made this function. But while reviewing I don't remember the specific purpose.